src/scancode/cli.py

```diff
@@ -185,6 +186,11 @@ def validate_formats(ctx, param, value):
         raise click.BadParameter('Invalid template file: "%(value)s" does not exists or is not readable.' % locals())
     return value
 
+def ignore_files(ctx, param, value):
+    for file in value.split(','):
+        user_ignored.update({
```

Comment:
Ah, You need to re-push your branch.... https://github.com/yashdsaraf/scancode-toolkit/commit/6401c5b406be5e6bcfb8b2c3246eae14d3782bc7 is not showing up in this PR. Hence all my comments because things here are still looking like you had made no changes :)